src/bench/prevector.cpp

```diff
@@ -27,35 +27,35 @@ template <typename T>
 static void PrevectorDestructor(benchmark::Bench& bench)
 {
     bench.batch(2).run([&] {
-        prevector<28, T> t0;
-        prevector<28, T> t1;
-        t0.resize(28);
-        t1.resize(29);
+        prevector<36, T> t0;
+        prevector<36, T> t1;
+        t0.resize(36);
+        t1.resize(37);
```

Comment:
maybe a "stupid" question, but why not use a constant for the 36 in this file too?

Reply:
I could, of course, for now I was going for simplifying the diff, not make the code more readable. I can of course do that later, but for now I'm looking for the simplest diff which demonstrates the problem and the proposed solution, and after I'm sure there's general support for it, we can sneak in code refactors as well.

Reply:
Thanks, extracted to a separate commit - as suggested by @luke-jr as well - added you both as co-authors.